modules/installation-initializing.adoc

```diff
@@ -219,9 +205,6 @@ endif::ibm-cloud[]
 ifdef::osp[]
 {rh-openstack-first}.
 endif::osp[]
-ifdef::vsphere,vmc[]
-VMware vSphere.
-endif::vsphere,vmc[]
```

Comment:
Did you mean to only delete the `vmc` parameter here?

Reply:
Thanks for the review, @bscott-rh . Yes I did. Do you know of a reason as why I should keep this parameter?

Reply:
Thanks for the review, @bscott-rh . Well spotted. I corrected re-added back in

```
ifdef::vsphere[]
VMware vSphere.
endif::vsphere[] 
```